mxd/backend-service/src/main/java/org/eclipse/tractusx/mxd/backendservice/store/SqlContentStoreServiceImpl.java

```diff
@@ -88,30 +88,28 @@ public StoreResult<ContentResponse> findById(String contentId) {
 
     @Override
     public String save(Object content) {
-        String urlResponse = "";
         try (var connection = getConnection()) {
-            urlResponse = insertInternal(connection, content).toString();
-
+            String urlResponse = insertInternal(connection, content).toString();
+            return urlResponse;
```

Comment:
Return directly instead of initializing into a variable and return it.

Reply:
Updated